### DISCUSSION TITLE
PR Commits do not live update

### DISCUSSION THREAD

**Discussion by pzcfg** [community/community#11633] [Pull Requests]
If you create a PR and view the commits, the counts will live update as you push subsequent commits, but the actual list of commits will not update. I tried switching between tabs but it still only shows the commits that were there when the page was loaded. Only refreshing the whole page got the new commits to show up.

![vHjMVYc](https://user-images.githubusercontent.com/94077014/154370306-d2bc142c-f243-41aa-9607-a557bfc9c888.png)

Repro Steps: Create a PR, view the PR's commit tab, push some additional commits, then navigate between the tabs.

**Comment by willsmythe** [comment]
Thanks for reporting. We have an internal issue tracking it.
